packages/backend-modules/access/lib/grants.js

```diff
@@ -305,7 +318,7 @@ const request = async (granter, campaignId, payload, t, pgdb, redis, mail) => {
 
   await eventsLib.log(grant, 'request', pgdb)
 
-  const perks = await grantPerks(grant, granter, campaign, t, pgdb, redis, mail)
+  const perks = grantPerks(grant, granter, campaign, t, pgdb, redis, mail)
```

Comment:
```suggestion
  const perks = await grantPerks(grant, granter, campaign, t, pgdb, redis, mail)
```

We do need to resolve that Promise before we continue.